packages/app/models/token/types.ts

```diff
@@ -1,10 +1,19 @@
-export interface Token {
+import { TokenSubgraph } from "@stackly/sdk";
+
+type TokenWithoutIdAndAddress = Omit<TokenSubgraph, "id" | "address">;
+
+interface TokenWithAddress extends TokenWithoutIdAndAddress {
   address: string;
-  decimals: number;
-  name: string;
-  symbol: string;
 }
 
+interface TokenWithId extends TokenWithoutIdAndAddress {
+  address: string;
+}
+
+type TokenWithAddressAndId = TokenWithAddress & TokenWithId;
```

Comment:
This type is irrelevant. as it's the same as `TokenSubgraph`

Reply:
true